src/js/stores/__tests__/BaseStore-test.js

```diff
@@ -0,0 +1,65 @@
+jest.dontMock('../BaseStore');
+
+var BaseStore = require('../BaseStore');
+
+describe('BaseStore', function () {
+
+  beforeEach(function () {
+    this.instance = new BaseStore();
+  });
+
+  describe('#get', function () {
+
+    it('should return undefined if no key is given', function () {
+      expect(this.instance.get()).toEqual(null);
+    });
+
+    it('should return undefined if given an object', function () {
+      expect(this.instance.get({})).toEqual(null);
+    });
+
+    it('returns null if property hasn\'t been defined', function () {
+      expect(this.instance.get('foo')).toEqual(null);
```

Comment:
Shouldn't it blow up in this case? In a sense the store is lying to us. If I'm accessing a property that was never defined, I'd like to know that rather than having the store tell me that property is defined and set to `null`.
What do you think?